### ISSUE TITLE
HTTP Request error path does not work

### ISSUE THREAD

**Issue by TitaniumCoder477** [n8n-io/n8n#11518]
### Bug Description

The "HTTP Request" error branch does not work.

### To Reproduce

1. Create an HTTP Request
2. Configure Settings > On Error > Continue (using error output)
3. Test with website unreachable
4. Instead of activating whatever is on the Error branch, a popup in the bottom right corner just says, "Problem running workflow. Request failed with status code 504."

### Expected behavior

I am attempting to use the "HTTP Request" to monitor a website. When the website responds, nothing should happen because I intentionally have no outcome on the Success branch. When the website fails to respond (i.e. 504), I expect an email to be sent to me because I have "Send Email" on the Error branch. However, no matter how I configure the Settings, I cannot get this to work. I just get that popup in the bottom right corner that says, "Problem running workflow. Request failed with status code 504."

### Operating System

Debian 6.1.112-1

### n8n Version

1.65.2

### Node.js Version

Whatever version comes with the Docker package

### Database

SQLite (default)

### Execution mode

main (default)

**Comment by Joffcom** [comment]
Hey @TitaniumCoder477,

We have created an internal ticket to look into this which we will be tracking as "GHC-382"

**Comment by TitaniumCoder477** [comment]
Sounds good, thanks. I also just noticed that if the workflow is activated and the site is down, n8n becomes unresponsive! For example, I can't get the Home page to load, nor the All executions page.

**Comment by wodor** [comment]
interesting I'm on 1.65.2 too and this works fine

![image](https://github.com/user-attachments/assets/79a9287a-4c3c-4036-bed3-8779f44280b7)

**Comment by TitaniumCoder477** [comment]
So I think my problem may be self-inflicted. Both the n8n server and the webserver I am attempting to monitor are hosted on a dedicated VLAN. Both have Internet out, but Internet in is via a reverse proxy in the cloud which is connected to my on-prem router by an IPsec tunnel. When I disconnected the VPN (i.e. simulating an outage), BOTH the n8n server AND the webserver are unreachable from the Internet. That is, their public DNS routes to the reverse proxy which cannot reach the servers because the VPN is down. This explains why n8n becomes unresponsive when I drop the VPN. But I wasstill uncertain why n8n, which still has Internet out, would not be able to (1) notice the webserver is unreachable and (2) send an email. However, now that I am on the trail of what is really going on, I looked back through the execution logs and found that when the webserver was unreachable, n8n did in fact try to send an email but received "Invalid login: 535 5.7.8 Authentication credentials invalid." I guess this is par for the course this week! Sorry for the hassle I may have caused!

**Comment by Joffcom** [comment]
@TitaniumCoder477 sounds like we can probably close this one then.

**Comment by TitaniumCoder477** [comment]
Yes, sorry. I should have closed it with my previous comment! Thank you!
